app/controllers/analytics_controller.rb

```diff
@@ -3,9 +3,9 @@
 # Controller for analytics page. A lot of reporting going on.
 # TODO: Some of these reports may need to be asynchronous as the data grows
 class AnalyticsController < ApplicationController
-  before_action :check_for_admin_or_analyst, only: %i[show index]
```

Comment:
Honestly, there is no show action so you can remove any references to that.

Reply:
I actually remove that line completely because the `before_action` was doing exactly what the policy will be doing